prepare_release.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+import argparse
+import re
+import subprocess
+
+
+def run_towncrier(tag):
+    cmd = ("towncrier", "build", "--version", tag.strip("v"))
+
+    return subprocess.call(cmd)
```

Comment:
does not seem to be using towncrier so far

Reply:
I opted for leaving the script as it is but commented out the call to this function in the `__main__` block.